### PR TITLE
added my basil haydens viewer thing

### PR DESCRIPTION
**Pull request by AbdellaToronto**
Resolves story: https://www.pivotaltracker.com/story/show/139908651

### CODE REVIEW

**Comment by AbdellaToronto** [comment]
Yo Bro, your PR is bumpin'

File: src/app/app.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-
+import Home from '../containers/home';
 function App() {
   return (
     <div>
-      Hello world!
+      <Home />
```
Comment (AbdellaToronto) [inline src/app/app.js:6]: What happened to Hello world?

File: src/reducers/index.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { combineReducers } from 'redux';
 import { routerReducer } from 'react-router-redux';
 import { reducer as formReducer } from 'redux-form';
+import DrinkReducer from './selected-drink';
 
 const rootReducer = combineReducers({
-  routing: routerReducer,
```
Comment (AbdellaToronto) [inline src/reducers/index.js:6]: Great reordering!

**Review by AbdellaToronto — COMMENTED** [review]
Great work! I had some thoughts about your naming convention, maybe we can talk about that later

**Review by alexxswanson — COMMENTED** [review]
I am reviewing this

**Review by alexxswanson — APPROVED** [review]
approved

**Pull request merged**